### DISCUSSION TITLE
Is there any way to add an id to the canvas element

### DISCUSSION THREAD

**Discussion by liveman1** [openseadragon/openseadragon#2119] [Q&A] [answered]
Hi,
it would be great to have an 'id' in the canvas element.
Is there any way to define one or is it necessary to add it in the openseadragon.js file?

Thanks a lot!

**Comment by iangilman** [comment, marked as answer]
There's no mechanism for that in the API, but you can do `viewer.drawer.canvas.id = 'foo'` or something like that.

**Comment by liveman1** [comment]
Thanks a lot! That works!
